pages/public_cloud/ai_machine_learning/deploy_tuto_17_streamlit_whisper/guide.en-gb.md

```diff
@@ -0,0 +1,306 @@
+---
+title: AI Deploy - Tutorial - Deploy Whisper Speech Recognition Model
+excerpt: How to deploy OpenAI’s Whisper Speech Recognition Model
+updated: 2023-12-12
+---
+
+> [!primary]
+>
+> AI Deploy is covered by **[OVHcloud Public Cloud Special Conditions](https://storage.gra.cloud.ovh.net/v1/AUTH_325716a587c64897acbef9a4a4726e38/contracts/d2a208c-Conditions_particulieres_OVH_Stack-WE-9.0.pdf)**.
+>
+
+## Introduction
+
+OpenAI's Whisper large-v3 is the latest iteration of the open-source Whisper Automatic Speech Recognition (ASR) model. 
+
+Released on November 6, 2023, Whisper large-v3 has gained recognition in various benchmarks as the top-performing automatic speech recognition (ASR) model. This is why it represents the state of the art in the ASR field. More generally, Whisper has become a powerful tool for transcribing speech in approximately 100 languages, including non-english speech transcription and translation into English.
+
+![Overview](images/speech-to-text.png){.thumbnail}
+
+## Objective
+
+In this tutorial, we will guide you through the process of deploying the Whisper large-v3 model in a simple application on AI Deploy for production use. 
+
+Unlike a locally hosted application, using AI Deploy offers extremely fast inference since it is deployed on powerful resources (GPUs), and other features like sharing capabilities. You will also learn how to build and use a custom Docker image as part of the deployment process.
```

Comment:
I would have added the link of the guide about "How to create custom Docker image" on custom **custom Docker image**